.github/workflows/documentation-preview-cleanup-request.yml

```diff
@@ -0,0 +1,20 @@
+name: Documentation Preview Cleanup Request
+
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  on-close:
+    runs-on: ubuntu-latest
```

Comment:
minor thing, I would suggest to set the version instead of latest to be sure about it [ubuntu-24.04](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md)

this would apply to the other files as well

Reply:
Thanks, I've updated all workflows to use `ubuntu-24.04`